PSReadLine/PSReadLine.csproj

```diff
@@ -5,9 +5,9 @@
     <RootNamespace>Microsoft.PowerShell.PSReadLine</RootNamespace>
     <AssemblyName>Microsoft.PowerShell.PSReadLine2</AssemblyName>
     <NoWarn>$(NoWarn);CA1416</NoWarn>
-    <AssemblyVersion>2.2.0.0</AssemblyVersion>
-    <FileVersion>2.2.0</FileVersion>
-    <InformationalVersion>2.2.0-beta3</InformationalVersion>
+    <AssemblyVersion>2.3.0.0</AssemblyVersion>
+    <FileVersion>2.3.0</FileVersion>
+    <InformationalVersion>2.3.0-beta1</InformationalVersion>
```

Comment:
Can you please revert changes in this file? The next release will be 2.2.0-beta3